src/eva/eva_base.py

```diff
@@ -133,7 +133,7 @@ def eva(eva_config, eva_logger=None):
     # Get the list of applications
     try:
         diagnostic_configs = eva_dict['diagnostics']
-    except Exception:
+    except Exception KeyError:
```

Comment:
Does this run? I was suggesting `except KeyError:`

Reply:
No, I made a typo. Will remove